### ISSUE TITLE
[SampleCard-StockUpdate; Config-testVariantHostConfig.json]: The color contrast ratio for the text "'Open', 'High', 'Low'"is 3.24 does not meet requirement of at least 4.5:!

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/AdaptiveCards#7386]
https://microsoft.visualstudio.com/OS/_workitems/edit/37992208
